code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -244,6 +244,20 @@
 		"<span class='userdanger'>You feel a powerful shock coursing through your body!</span>", \
 		"<span class='italics'>You hear a heavy electrical crack.</span>" \
 		)
+	if(iscarbon(pulling) && !illusion && source != pulling)
+		if(ishuman(pulling))
+			var/mob/living/carbon/human/H = pulling
+			H.electrocute_act(shock_damage*0.75, src, 1, safety, override, tesla_shock, illusion, stun)
+		else
+			var/mob/living/carbon/C = pulling
+			C.electrocute_act(shock_damage*0.75, src, 1, safety, override, tesla_shock, illusion, stun)
+	if(iscarbon(pulledby) && !illusion && source != pulledby)

```

Comment:
source can never be pulledby if it's a carbon since source is typed as an object